svetlanagorchanina/nodejs-aws-be
# Task 9

**Main tasks**

**Additional (optional) tasks**
How to test:
Get products list
Create new product
Get products list - result shouldn’t have new product
Wait more than 2 minutes
Get products list - result should have new product

---

Application link: https://d12r4l4twrn36b.cloudfront.net/

Cart API: https://edvg921hw8.execute-api.eu-west-1.amazonaws.com/dev/api/profile/cart
BFF Cart API: http://svetlanagorchanina-bff-api-develop-proxy.eu-west-1.elasticbeanstalk.com/cart/api/profile/cart

Product API: https://fgkaarac1j.execute-api.eu-west-1.amazonaws.com/dev/products
BFF Product API: http://svetlanagorchanina-bff-api-develop-proxy.eu-west-1.elasticbeanstalk.com/product/products

---

#### Notes:

Product can be created via "Create product" button (https://d12r4l4twrn36b.cloudfront.net/admin/products)
or 
```
fetch("https://fgkaarac1j.execute-api.eu-west-1.amazonaws.com/dev/products", {
  "headers": {
    "accept": "application/json, text/plain, */*",
    "content-type": "application/json;charset=UTF-8",
  },
  "body": "{\"count\":34,\"price\":12,\"description\":\"Test\",\"title\":\"Test\"}",
  "method": "POST",
});
```
